README.md

```diff
@@ -49,7 +49,7 @@ app.post('/graphql', graphqlHTTP({
 
 app.get('/graphql', graphqlHTTP({
   schema: MyGraphQLSchema,
-  graphiql: true // set it to false to disable the GraphiQL interface
```

Comment:
I think the odds of confusion are extremely unlikely, and the downside even in the unlikely event of a misconfiguration is minimal, so I feel pretty happy about reducing the clutter in the example.